bindings/c/test/apitester/TesterBlobGranuleCorrectnessWorkload.cpp

```diff
@@ -168,7 +175,8 @@ class ApiBlobGranuleCorrectnessWorkload : public ApiWorkload {
 				    }
 			    }
 			    schedule(cont);
-		    });
+		    },
+		    false);
```

Comment:
My intention here was to just use the retry loop on retriable errors like transaction_too_old, future_version, etc..
If there is a better way to do that I can change it.

Reply:
The failOnError flag does not affect the retry logic. It is just specifies whether the workload should be considered as failed and stopped in case of a non-retriable error. It means that the test expect an error to be produced. In that case it would make sense if the error code is as expected and fail otherwise. Otherwise you risk that the test will be green no matter what happens.